tests/attacks/property_inference_test.py

```diff
@@ -23,4 +22,9 @@ def test_property_inference_attack():
     target_model = Classifier._to_art_classifier(model, num_classes, input_shape)
 
     attack = PropertyInferenceAttack(target_model, train_dataset)
-    attack.attack()
+    assert attack.input_shape == input_shape, f"Wrong input shape. Input shape should be {input_shape}."
+    assert attack.amount_sets >= 2 and attack.amount_sets % 2 == 0, "Number of shadow classifiers must be even and greater than 2."
```

Comment:
Message is not 100% correct: Number of shadow classifiers must be >= 2, not >2

Reply:
assert will raise if amount_sets is not even and amount_sets < 2. So this should be correct